src/main/java/com/coherentsolutions/pot/insurance/controller/EmployeeController.java

```diff
@@ -23,6 +27,14 @@
 public class EmployeeController {
     private final EmployeeService employeeService;
 
+    @GetMapping("/filtered")
+    @ResponseStatus(HttpStatus.OK)
+    public Page<EmployeeDTO> filterAndSortEmployees(
```

Comment:
Let's use the same approach in the naming of controllers method
```suggestion
    public Page<EmployeeDTO> getFilteredSortedEmployees(
```

Reply:
updated namings